pwa/src/components/TestResultsForm.tsx

```diff
@@ -0,0 +1,122 @@
+        { value: 1, label: 'Positive' },
+    ]
+    const uaResultElements: ReactElement[] = []
+
+    return (
+        <div
+            id="test-results-form-wrapper"
+            data-testid="test-results-form-wrapper"
+        >
+            <List>
+                {' '}
+                <CheckboxItem
+                    id="pregnancy-test"
+                    data-testid="pregnancy-test"
+                    onChange={() => setPregnancyTest(pregnancyTest ? 0 : 1)}
+                    title="HCG (Pregnancy) Test Performed"
+                >
+                    HCG (Pregnancy) Test Performed
+                </CheckboxItem>
+                {pregnancyTest ? (
+                    <div id="pregnancy-results" data-testid="pregnancy-results">
+                        <List renderHeader={() => 'HCG (Pregnancy) Results'}>
+                            {radioPosNeg.map((i) => (
+                                <RadioItem
+                                    key={i.value}
+                                    data-testid={
+                                        `pregnancy-results-` +
+                                        i.label.toLowerCase()
+                                    }
+                                    name={
+                                        `pregnancy-results-` +
+                                        i.label.toLowerCase()
+                                    }
+                                    // name="pregnancy-results-radio"
+                                    checked={pregnancyTestPositive === i.value}
+                                    onClick={() =>
+                                        setPregnancyTestPositive(i.value)
+                                    }
+                                >
+                                    {i.label}
+                                </RadioItem>
+                            ))}
+                            <WhiteSpace className="list-whitespace" size="lg" />
+                        </List>
+                    </div>
+                ) : null}
+                <CheckboxItem
+                    id="blood-sugar-test"
+                    data-testid="blood-sugar-test"
+                    onChange={() => setBloodSugarTest(bloodSugarTest ? 0 : 1)}
+                >
+                    Blood Sugar Test Performed
+                </CheckboxItem>
+                {bloodSugarTest ? (
+                    <div
+                        id="blood-sugar-results"
+                        data-testid="blood-sugar-results"
+                    >
+                        <List renderHeader={() => 'Blood Sugar Results'}>
+                            <InputItem
+                                type="number"
+                                id="blood-sugar-results-val"
+                                data-testid="blood-sugar-results-val"
+                                placeholder="mg/dL" // TOOD: unsure if the unit is mg/dL or mmol/L, could leave this out
```

Comment:
Unit for this field should be determined or this placeholder should be removed.

Reply:
I vote keeping it as mg/dl for now. Can be confirmed with a CHI rep at a later date.

Reply:
Do you recommend leaving the TODO tag there or removing and creating an issue?

Reply:
Go ahead and make an issue, that's a good point.

Reply:
Created #107